### PR TITLE
Adding resumeSchedulerAndAction(bool) and pauseSchedulerAndActions(bool)

### PR DESCRIPTION
**Pull request by gelldur**
request from http://www.cocos2d-x.org/boards/6/topics/6887

### CODE REVIEW

File: cocos2dx/base_nodes/CCNode.cpp [hunk 1]
```diff
@@ -1094,12 +1094,41 @@ void CCNode::resumeSchedulerAndActions()
     m_pActionManager->resumeTarget(this);
 }
 
+void CCNode::resumeSchedulerAndActions(bool applayToChildren)
```
Comment (dumganhar) [inline cocos2dx/base_nodes/CCNode.cpp]: `applaytoChildren` ~> `applyToChildren`?

**Comment by gelldur** [comment]
I fixed misspell.

**Comment by minggo** [comment]
Is it needed? I think the sub class can override this method if it needs this feature.

**Comment by gelldur** [comment]
Hmm i use it in CCLayer and in CCScene. Maybe it is'n so much needed but it is short method and nice feature to make pause/resume game

**Comment by gelldur** [comment]
Merge or don't merge :)

**Pull request closed without merging**